cmd/av/stack_split.go

```diff
@@ -0,0 +1,114 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"emperror.dev/errors"
+	"github.com/aviator-co/av/internal/git"
+	"github.com/aviator-co/av/internal/utils/colors"
+	"github.com/go-git/go-git/v5/plumbing"
+	"github.com/spf13/cobra"
+)
+
+var stackSplitCmd = &cobra.Command{
+	Use:          "split",
+	Short:        "Split the last commit of the current branch into a new branch",
+	SilenceUsage: true,
+	Args:         cobra.ExactArgs(1),
```

Comment:
Can we make this support 0 args as well and try using the same logic as in `stack branch-commit` to generate a branch name if none is supplied?